buzzmobile/launch/simulation.launch

```diff
@@ -9,7 +9,7 @@
         <include file="$(find buzzmobile)/launch/includes/mapping.launch"/>
         <include file="$(find buzzmobile)/launch/includes/steering.launch"/>
         <include file="$(find buzzmobile)/launch/includes/lidar.launch"/>
-        <include file="$(find buzzmobile)/launch/includes/controller.launch"/>
+        <!--<include file="$(find buzzmobile)/launch/includes/controller.launch"/>-->
```

Comment:
huh i guess this change makes sense, right? we don't want controller in simulation

wait controller.launch also has car_pose_mux, which we need to make this work. how did you test this at all without car_pose_mux?